study.md

```diff
@@ -67,5 +70,10 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+The client makes a GET request, the router receives what the client wants and
+passes that information to the controller; the controller doesn't really care
```

Comment:
The controller does care about the information because it may not perform an action depending on the parameters being passed in.